markdown/dev/howtos/ways-to-contribute/technical-writing/patterns/en.md

```diff
@@ -2,6 +2,13 @@
 title: Writing for freesewing.org
 ---
 
-You could write documentation for freesewing.org, our makers website.
+A good way of contributing to FreeSewing is writting documentation! 
```

Comment:
Pet peeve: Exclamation marks are not for conveying excitement. They are to indicate somebody is yelling.